hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/validators/block/TransactionRecordParityValidator.java

```diff
@@ -72,10 +72,10 @@ public boolean appliesTo(@NonNull final HapiSpec spec) {
 
     public static void main(String[] args) throws IOException {
         final var input =
-                "/Users/michaeltinker/AlsoDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/data/block-streams/block-0.0.3/";
+                "/Users/michaeltinker/YetAnotherDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/data/block-streams/block-0.0.3/";
         final var blocks = BlockStreamAccess.BLOCK_STREAM_ACCESS.readBlocks(Paths.get(input));
         final var loc =
-                "/Users/michaeltinker/AlsoDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/data/recordStreams/record0.0.3";
+                "/Users/michaeltinker/YetAnotherDev/hedera-services/hedera-node/test-clients/build/hapi-test/node0/data/recordStreams/record0.0.3";
         final var records = RecordStreamAccess.RECORD_STREAM_ACCESS.readStreamDataFrom(loc, "sidecar");
```

Comment:
Commented this in a prior PR too.
This could be done with no placeholder as follows:
```java
        final Path nodeZeroDataFolder =
                WorkingDirUtils.workingDirFor(0, "hapi").resolve("data").toAbsolutePath().normalize();
        final String blockFolder = nodeZeroDataFolder.resolve("block-streams/block-0.0.3").toString();
        final String recordFolder = nodeZeroDataFolder.resolve("recordStreams/record0.0.3").toString();
```
Then use the two strings to get `blocks` and `records`.

Reply:
Ah good point, fixed.